### PR TITLE
Add AutoRespawn

### PR DESCRIPTION
**Pull request by SleepingPig35**
## Summary by CodeRabbit

- **New Features**
    - Introduced an "AutoRespawn" module that automatically respawns the player after death with a configurable delay setting.

### CODE REVIEW

**Comment by TejasLamba2006** [comment]
Hello @SleepingPig35,

Would you be able to incorporate a slider to adjust the respawn delay time after death? This addition would offer more configuration options for the client.

**Comment by SleepingPig35** [comment]
Okay. I'll add that.

**Review by TejasLamba2006 — CHANGES_REQUESTED** [review]
Add delay

**Comment by SleepingPig35** [comment]
Done sorry lol I went to do some chores

File: src/main/java/keystrokesmod/module/impl/other/AutoRespawn.java [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+        this.registerSetting(new DescriptionSetting("Automatically respawns you after you die."));
+        this.registerSetting(delay = new SliderSetting("Delay (ms)", 0, 0, 100000, 1000));
+    }
+   @SubscribeEvent
+    public void onReceive(@NotNull ReceivePacketEvent event) {
+        if(Minecraft.getMinecraft().thePlayer.isDead){
+            try {
+                Thread.sleep((long) delay.getInput());
+            } catch (InterruptedException e) {
+                e.printStackTrace();
```
[the remark this answers is not in the document]
Comment (SleepingPig35) [reply src/main/java/keystrokesmod/module/impl/other/AutoRespawn.java:27]: No point lol this never happens

File: src/main/java/keystrokesmod/module/impl/other/AutoRespawn.java [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+            }
+            PacketUtils.sendPacketNoEvent(new C16PacketClientStatus(C16PacketClientStatus.EnumState.PERFORM_RESPAWN));
+        }
+    };
```
[the remark this answers is not in the document]
Comment (SleepingPig35) [reply src/main/java/keystrokesmod/module/impl/other/AutoRespawn.java]: Added.

**Comment by xia-mc** [comment]
Hey, can you use ```ScheduledExecutorService``` instand of create a new thread?
like this:
```java
Raven.getExecutor().schedule(Runnable action, long time, TIMEUNIT timeUnit);
```

**Pull request merged**